doc/sphinx-guides/source/versions.rst

```diff
@@ -0,0 +1,30 @@
+.. _guides_versions:
+
+Dataverse Guides Versions
+=========================
+
+- 4.8.6 *latest*
+- `4.8.5 </en/4.8.5/>`__
```

Comment:
This is another file that has to be updated at release time so this page should be updated: http://guides.dataverse.org/en/4.8.6/developers/making-releases.html#bump-version-numbers

Reply:
Going to remove the "latest" label in order be more generic and not apply a label that will be outdated on the next release.

Reply:
After discussing the use of this page with Phil in more detail, he proposed it might not be necessary if we instead link to the GitHub releases view (https://github.com/IQSS/dataverse/releases), and edit each of the release notes with a documentation link.